user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr :hoge
+  def initialize
+    @hoge =    [a,b,c,d]*''
```

Comment:
__[development][rubocop]__
* Operator `=` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Unnecessary spacing detected.
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Favor `Array#join` over `Array#*`. [:ref](https://github.com/bbatsov/ruby-style-guide#array-join)
* Surrounding space missing for operator `*`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->